spec/system/developers_search_form_system_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+describe 'Search for developer' do
+  before do
+    test_developer_one = create(:developer, email: 'tester123@gmail.com')
```

Comment:
@truongle7793 

<img width="701" alt="screen shot 2018-04-04 at 14 12 58" src="https://user-images.githubusercontent.com/4110469/38297725-945723b4-381f-11e8-80ee-41300e1184b5.png">

Reply:
@thienhv 
Thanks!
I have changed it, it look much cleaner now.